.github/renovate.json

```diff
@@ -2,8 +2,11 @@
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "dependencyDashboard": true,
   "suppressNotifications": ["prEditedNotification"],
-  "extends": ["config:base"],
+  "extends": ["config:recommended"],
   "labels": ["bot: dependencies"],
+  "semanticCommits": "disabled",
+  "separateMajorMinor": false,
```

Comment:
We already have this setting for all of our dependency groups; this just moves that setting to the top level instead